test/unit/asset-class-constructor-directory/input.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs")
+const path = require("path")
+
+class Foobar {
+  constructor(packagePath) {
+    console.log(fs.readFileSync(path.join(packagePath, 'asset1.txt'), 'utf8'))
```

Comment:
I tried changing `readFileSync` to `require` and that doesn't work either. So there must be something different about the npm package that made it work 🤔